### PR TITLE
Changed 'invite a friend' to 'add members'. Removed 'account' tab and…

### PR DESCRIPTION
**Pull request by siepra**
… corrected modal title

### CODE REVIEW

File: packages/ZbayLite/src/renderer/containers/widgets/settings/SettingsModal.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { useModal } from '../../hooks'
 const useSettingsModalData = () => {
   const data = {
     modalTabToOpen: useSelector(appSelectors.currentModalTab),
-    user: 'settingsModalUser',
+    user: useSelector(identity.selectors.currentIdentity).zbayNickname,
```
Comment (EmiM) [inline packages/ZbayLite/src/renderer/containers/widgets/settings/SettingsModal.tsx:13]: This throws error when you open the app for the first time (cause there is no identity yet)
`Cannot read property 'zbayNickname' of undefined`

**Comment by siepra** [comment]
Huh, I think it's a bigger problem. All the modals exists before we
initialize community data so this error will return whenever we use some
'fragile' selectors inside them. The question is will that be common
scenario or is it just a single case?

pon., 27 gru 2021, 13:35 użytkownik breadhunter ***@***.***>
napisał:

File: packages/ZbayLite/src/renderer/containers/widgets/settings/SettingsModal.tsx (hunk 1 above)
Comment (siepra) [reply packages/ZbayLite/src/renderer/containers/widgets/settings/SettingsModal.tsx:13]: Huh, I think it's a bigger problem. All the modals exists before we
initialize community data so this error will return whenever we use some
'fragile' selectors inside them. The question is will that be common
scenario or is it just a single case?

Comment (siepra) [reply packages/ZbayLite/src/renderer/containers/widgets/settings/SettingsModal.tsx:13]: Also, you shouldn't be able to reach that point without community at all

**Pull request merged**